agglayer/grpc/conversion_to_agglayer.go

```diff
@@ -0,0 +1,298 @@
+package grpc
+
+// This file contains the function to convert from grpc to agglayer types.
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	v1nodetypes "buf.build/gen/go/agglayer/agglayer/protocolbuffers/go/agglayer/node/types/v1"
+	v1types "buf.build/gen/go/agglayer/interop/protocolbuffers/go/agglayer/interop/types/v1"
+	agglayertypes "github.com/agglayer/aggkit/agglayer/types"
+	"github.com/agglayer/aggkit/bridgesync"
+	treetypes "github.com/agglayer/aggkit/tree/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var (
+	ErrNotImplemented        = errors.New("aggsender-validator not implemented")
```

Comment:
I think you don't need to export these errors.

```suggestion
	errNotImplemented        = errors.New("aggsender-validator not implemented")
	errNilCertificate            = errors.New("aggsender-validator nil certificate")
```

Reply:
We agree to expose errors as part of the interface